src/ConstructiveSolidGeometry/plotting/SurfacePrimitives/ConalPlane.jl

```diff
@@ -1,6 +1,6 @@
 function get_plot_points(c::ConalPlane{T}; n = 30) where {T <: AbstractFloat}
     v = get_vertices(c)
-    [Vector{CartesianPoint{T}}([v[1], v[2]]), Vector{CartesianPoint{T}}([v[2], v[3]]), Vector{CartesianPoint{T}}([v[3], v[4]]), Vector{CartesianPoint{T}}([v[4], v[1]])]
+    [Vector{CartesianPoint{T}}([v[1], v[2]]), Vector{CartesianPoint{T}}([v[2], v[4]]), Vector{CartesianPoint{T}}([v[3], v[4]]), Vector{CartesianPoint{T}}([v[3], v[1]])]
```

Comment:
This is intended?

Reply:
Yes, I changed the order that vertices are returned by `get_vertices(c)` when I improved the meshing of ConalPlane. I caught this extra to adjustment now